modules/dynamic-provisioning-about.adoc

```diff
@@ -5,21 +5,27 @@
 [id="about_{context}"]
 = About dynamic provisioning
 
-The StorageClass resource object describes and classifies storage that can 
+The StorageClass resource object describes and classifies storage that can
 be requested, as well as provides a means for passing parameters for
-dynamically provisioned storage on demand. StorageClass objects can also 
-serve as a management mechanism for controlling different levels of 
+dynamically provisioned storage on demand. StorageClass objects can also
+serve as a management mechanism for controlling different levels of
 storage and access to the storage. Cluster Administrators (`cluster-admin`)
- or Storage Administrators (`storage-admin`) define and create the 
-StorageClass objects that users can request without needing any intimate 
+ or Storage Administrators (`storage-admin`) define and create the
+StorageClass objects that users can request without needing any intimate
 knowledge about the underlying storage volume sources.
 
-The {product-title} persistent volume framework enables this functionality 
-and allows administrators to provision a cluster with persistent storage. 
-The framework also gives users a way to request those resources without 
+The {product-title} persistent volume framework enables this functionality
+and allows administrators to provision a cluster with persistent storage.
+The framework also gives users a way to request those resources without
 having any knowledge of the underlying infrastructure.
 
 Many storage types are available for use as persistent volumes in
 {product-title}. While all of them can be statically provisioned by an
 administrator, some types of storage are created dynamically using the
 built-in provider and plug-in APIs.
+
+When using Red Hat OpenShift Container Storage, the dynamic StorageClasses are
+created when Red Hat OpenShift Container Storage 4.2 is deployed from the
+Operator Hub as described in the
+link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.2/html-single/deploying_openshift_container_storage/index#verify_that_the_storage_classes_are_created_and_listed[Verify that the storage classes are created and listed]
+section.
```

Comment:
I don't think we should be adding OCS-specific documentation to this section for OCS as then we would need to include content for all other supported plug-ins. OCS content placement should make sense in the context of our docs, which it doesn't here.

My understanding from Ken is that the OCS content and links in OCP would be maintained on this page: https://docs.openshift.com/container-platform/4.2/storage/persistent_storage/persistent-storage-ocs.html